modules/farm_fd2_school/07-FD1/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>07 FD1 Tutorial</title>
+  </head>
+  <body>
+    <div id="fd1-tutorial">
+      <h1>07 FD1 Tutorial</h1>
+
+      <br />
+      <br /><br />
+
+      <label for="users">Users:</label>
+      <select
+        id="users"
+        v-model="selectedUser"
+      >
+        <option
+          v-for="user in userList"
+          v-bind:key="user.id"
+          v-bind:value="user"
+        >
+          {{user.name}}
+        </option>
+      </select>
+
+      <div
+        id="user-posts"
+        v-if="selectedUser"
+      >
+        <p>Posts by {{ selectedUser.name }}:</p>
+      </div>
+    </div>
+    <script src="https://unpkg.com/vue@3"></script>
+    <script>
+      const fd1Tutorial = Vue.createApp({
+        data() {
+          return {
+            userList: [],
+            selectedUser: null,
+          };
+        },
+        methods: {
+          async fetchUsers() {
+            const usersResponse = await fetch(
+              'https://jsonplaceholder.typicode.com/users'
+            );
+            const users = await usersResponse.json();
+            //console.log(users);
+            //const user = users[0];
+            //console.log(user["name"], user["username"], user["address"]["zipcode"], user["address"]["geo"]["lng"]);
+            //console.log(users[8]["name"]);
+            this.userList = users;
+          },
+        },
+        created() {
```

Comment:
The `created` lifecycle hook calls the `fetchUsers` method which is `async` so the `created` lifecycle hook should also be `async`.